bin/puppet_webhook

```diff
@@ -6,7 +6,13 @@ require 'webrick'
 require 'webrick/https'
 require 'puppet_webhook'
 
-config_file '../config.yml'
+if File.exist?('/etc/puppet-webhook/server.yml')
```

Comment:
I'd also like a command line argument which you can use to point it at.

Reply:
Do we want to add it as part of this PR or another one?

Reply:
Separate PR is fine.